build/release.mk

```diff
@@ -66,8 +66,22 @@ endif
 	@# Download prepackaged plugins
 	mkdir -p tmpprepackaged
 	@cd tmpprepackaged && for plugin_package in $(PLUGIN_PACKAGES) ; do \
-		curl -O -L https://plugins-store.test.mattermost.com/release/$$plugin_package.tar.gz; \
-		curl -O -L https://plugins-store.test.mattermost.com/release/$$plugin_package.tar.gz.sig; \
+		curl -f -O -L https://plugins-store.test.mattermost.com/release/$$plugin_package.tar.gz; \
+		curl -f -O -L https://plugins-store.test.mattermost.com/release/$$plugin_package.tar.gz.sig; \
+	done
+
+	@# Import Mattermost plugin public key
+	gpg --import build/plugin_publickey.asc
```

Comment:
If we import this a for-loop earlier, could we merge the two for loops and verify the signatures right after download?

Reply:
Also, would it make sense to match the name and encoding with what is in the `tests/` folder (i.e. name this `plugin-production-public-key.gpg`)?